.github/CODEOWNERS

```
@@ -0,0 +1,8 @@
+# Github code owners
+# Siehe auch https://help.github.com/articles/about-codeowners/
+#
+# Bitte die Sortierung dieser Datei beibehalten. Die zuletzt zutreffende Regel
+# hat Vorrang.
+
+# global codeowner
+* @andrecrema
```

```suggestion
* @andrecrema @tschreibereuropace 
```

Da es sich um eine API handelt die sowohl in Market Engine als auch im Dokservice eingebunden ist, wäre ich auch gern CodeOwner